Enable build on Visual Studio 2017

AndyGerlicher opened the pull request:
This will update things to work completely with only Visual Studio 2017 installed.

Tested with the RTW build to be released tomorrow on Windows 10 latest release (`14393.726`). Will update documentation as well to note the requirement for VS2017.

rainersigwald commented on netci.groovy:
```diff
@@ -50,6 +50,7 @@ def project = GithubProject
 
                         skipTestsWhenResultsNotFound = false
                     }
+					Utilities.setMachineAffinity(myJob, 'Windows_NT', 'latest-or-auto-dev15-rc')
```
Also update the `windowsScript` definition above to use dev15.

Or leave these changes out of this PR and send them as a followup PR (to `master`, since that's the current source of truth for our CI definitions).

KirillOsenkov replied:
This file seems to use spaces but you use tabs.

AndyGerlicher commented:
I updated the `cibuild.cmd` to not pass in that property. That should fix your issue @rainersigwald .

I also removed the `netci.groovy` changes and sent another PR.

AndyGerlicher commented:
@dotnet-bot test Windows_NT Build for Desktop please

rainersigwald commented:
@dotnet-bot test Windows_NT Build for Desktop please

rainersigwald commented:
@dotnet-bot test Windows_NT Build for Desktop please
